src/multicellular_stage/editor/MulticellularEditor.cs

```diff
@@ -304,6 +304,7 @@ protected override void ElapseEditorEntryTime()
     {
         // TODO: select which units will be used for the master elapsed time counter
         CurrentGame.GameWorld.OnTimePassed(1);
+        cellEditorTab.UpdateBackgroundImage(CurrentPatch.BiomeTemplate);
```

Comment:
Or if every single editor needs to implement this, then I think it might be worth considering putting this into the base editor class.

Reply:
I know, but the base class doesn't have `cellEditorTab` or `CurrentPatch`, I think it's not worth changing it for two lines of code multiplicated three times

Reply:
I was thinking more along the lines of the base class triggering a method about applying the current patch details to the environment after calling ElapseEditorEntryTime, that way we get this working "for free" without needing to add a bunch of extra logic to handle anything that might change in the current patch based on the elapsed time. The logic here would get pretty complex once sunlight can be reduced and the editor should show numbers calculated with the latest patch data. Though I guess that must already exists? Otherwise there's a bug. So I would actually recommend just making sure the latest image is also applied in that method.